### PR TITLE
Release notes for the 2.0.1 release

### PR DESCRIPTION
**Pull request by nacx**
This is the draft for the 2.0.1 release notes. The branch is pushed to the ASF remote, so feel free to commit directly to it to amend/fix/add/remove anything.

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://18a1cfe8f7d9cd8a50b0-32b7a8a1564e44899e8c2ab4210c7d12.r94.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://d362ae36369489084afc-a3c1bd6c15eec19677e3ba6c29d73325.r9.cf5.rackcdn.com/ to review your changes.

File: releasenotes/2.0.1.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* [Daniel Estevez](https://twitter.com/danielestevez) for his contributions to the Azure ARM load balancer APIs.
+* The Packet team for helping with the testing infrastructure.
+* Apache jclouds [PMC](http://people.apache.org/committers-by-project.html#jclouds-pmc) and community for verifying the release.
+* Check out who has been busy on [Open Hub](https://www.openhub.net/p/jclouds/contributors?query=&sort=latest_commit).
+
+## <a id="test"></a>Test Results
+
+Please see the discussion and the vote threads for live test results for 2.0.1:
+
+* RC1 [vote](https://s.apache.org/jclouds201rc1vote) and [discussion](https://s.apache.org/jclouds201rc1discuss) threads.
```
Comment (demobox) [inline releasenotes/2.0.1.md]: Like the shortlinks! ;-)

**Comment by jclouds-commentator** [comment]
Go to http://f192800187877c28d77d-912ab9cfe4c8304192c331e3bef81678.r92.cf5.rackcdn.com/ to review your changes.

File: releasenotes/2.0.1.md (hunk 1 above)
Comment (nacx) [inline releasenotes/2.0.1.md]: Yeah :) This way we don't need to update the site once the announcement is out!

**Comment by nacx** [comment]
rebuild please

**Comment by jclouds-commentator** [comment]
Go to http://972c5912208283f877ba-ba31b9d740060c9caae2e306155f3723.r3.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://bf5140d07fdc19ccf0af-3ea4ec4cfe7c2c6219f50049b124ff07.r51.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://4ab38f205d8808708833-58bff4ad454deb1b2850fca7b2db0169.r61.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://694dfff01a0761f42fb5-565cba4519a3b2e696b15868a3092da6.r86.cf5.rackcdn.com/ to review your changes.

**Pull request merged**